e2e/cypress/mockApis/wiremock.js

```diff
@@ -1,6 +1,9 @@
 const superagent = require('superagent');
 
-const url = 'http://localhost:9091/__admin';
+const host = process.env.WIREMOCK_BASE_URL || 'http://localhost:9091';
```

Comment:
allow overriding wiremock url as it will differ between running in docker compose (which uses docker networking) and outside it.